core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreConstants.java

```diff
@@ -67,7 +67,7 @@
                 "used by the font-end components for password validation");
         setProperty(passwordJavaRegExViolationErrorMsg, "Password RegEx Violation Error Message",
                 "Password pattern policy violated.", passwordJavaRegExViolationErrorMsgDescription);
-        setProperty("RolenameJavaRegEx", "Role Name RegEx (Java)", "^[\\S]{5,30}$", "A regular expression to validate role names");
+        setProperty("RolenameJavaRegEx", "Role Name RegEx (Java)", "[a-zA-Z0-9._\\-|//]{3,30}$", "A regular expression to validate role names");
```

Comment:
We need to check other user store managers as well. This is only for JDBCUserStoreManager.

Reply:
Both username RegEx and role Name RegEx in other user stores are according to the required regular expression. 
https://github.com/wso2/carbon-kernel/blob/0ba096f38a056321e82095a5fe0125283ae14919/core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ActiveDirectoryUserStoreConstants.java#L141

https://github.com/wso2/carbon-kernel/blob/0ba096f38a056321e82095a5fe0125283ae14919/core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadWriteLDAPUserStoreConstants.java#L111

Reply:
:+1: